config/internal/common/mlmd-envoy-dashboard-access-policy.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+kind: NetworkPolicy
```

Comment:
how does this get deployed? you will need to include it here: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/controllers/common.go#L23

Reply:
Done, updated here to add a call to the network policy template.